src/utils/material.js

```diff
@@ -10,16 +10,16 @@ module.exports.updateMap = function (shader, data) {
   var el = shader.el;
   var src = data.src;
   var material = shader.material;
-  var materialSystem = el.sceneEl.systems.material;
+  var textureSystem = el.sceneEl.systems.texture;
```

Comment:
The only reason I can think of is that a system is specific to a single A-Frame scene, and a utility module is global among all A-Frame scenes.

I'll keep everything in the material system then. I think that doesn't hurt.